### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

File: .github/workflows/upgrade-python-requirements.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: setup testeng-ci
         run: |
-          git clone https://github.com/edx/testeng-ci.git
+          git clone https://github.com/openedx/testeng-ci.git
```
Comment (feanil) [inline .github/workflows/upgrade-python-requirements.yml]: This repo did not get moved as a part of the transition and still lives in the `edx` org. We should not update this URL.

**Review by feanil — CHANGES_REQUESTED** [review]

File: .github/workflows/upgrade-python-requirements.yml (hunk 1 above)
Comment (sarina) [reply .github/workflows/upgrade-python-requirements.yml]: Thanks - I'm going to address this after I set up the automation to correctly update pylintrc

**Comment by sarina** [comment]
@feanil my script now auto-does edx_lint and I removed the only other occurrence I found of github.com/edx per your review. 

edit: i was spacing yesterday afternoon, i can continue my script without this merging - so take your time.

File: .github/workflows/upgrade-python-requirements.yml (hunk 1 above)
Comment (sarina) [reply .github/workflows/upgrade-python-requirements.yml]: The correct solution is to use the `.github/workflow-templates/upgrade-python-requirements.yml` file, which I've done now.

**Review by feanil — APPROVED** [review]

**Comment by aht007** [comment]
@sarina I can see that @openedx/arbi-bom was removed from the team-reviewers due to this transition but no other user/team was added for review. This would mean that requirements upgrade PR would go unnoticed. @openedx/2u-aperture can you please put yourself as the reviewing team in the workflow. Thanks

**Comment by sarina** [comment]
This is a pull request closed 2 months ago. I am confused why comments are being added.

**Pull request merged**